src/main/kotlin/org/mule/wsdl/parser/model/WsdlModel.kt

```diff
@@ -16,6 +17,12 @@ class WsdlModel(val location: String,
 
   fun getMessage(qname: QName) = messages.find { m -> qname.toString() == m.name }
 
-  fun getOperation(name: String): OperationModel? = services.flatMap { it.ports }.flatMap { it.operations }.firstOrNull { it.name == name }
+  fun getOperation(name: String): OperationModel? {
+    val operations = services.flatMap { it.ports }.flatMap { it.operations }
+    if (operations.size > 1) {
```

Comment:
Where are you checking against the name parameter?

Reply:
🤔